tensorflow-op-codegen/src/bin/eager.rs

```diff
@@ -0,0 +1,783 @@
+            Some(AttrValue_oneof_value::i(i)) => write!(w, "Some({}i64)", i)?,
+            Some(AttrValue_oneof_value::f(f)) => {
+                if f == &f32::INFINITY {
+                    write!(w, "Some(f32::INFINITY)")?;
+                } else if f == &f32::NEG_INFINITY {
+                    write!(w, "Some(f32::NEG_INFINITY)")?;
+                } else if f == &f32::NAN {
+                    write!(w, "Some(f32::NAN)")?;
+                } else {
+                    write!(w, "Some({}f32)", f)?;
+                }
+            }
+            Some(AttrValue_oneof_value::b(b)) => write!(w, "Some({})", b)?,
+            Some(AttrValue_oneof_value::field_type(t)) => match t.descriptor().name() {
+                "DT_FLOAT" => write!(w, "Some(crate::DataType::Float)")?,
+                "DT_DOUBLE" => write!(w, "Some(crate::DataType::Double)")?,
+                "DT_INT32" => write!(w, "Some(crate::DataType::Int32)")?,
+                "DT_UINT8" => write!(w, "Some(crate::DataType::UInt8)")?,
+                "DT_INT16" => write!(w, "Some(crate::DataType::Int16)")?,
+                "DT_INT8" => write!(w, "Some(crate::DataType::Int8)")?,
+                "DT_STRING" => write!(w, "Some(crate::DataType::String)")?,
+                "DT_COMPLEX64" => write!(w, "Some(crate::DataType::Complex64)")?,
+                "DT_INT64" => write!(w, "Some(crate::DataType::Int64)")?,
+                "DT_BOOL" => write!(w, "Some(crate::DataType::Bool)")?,
+                "DT_QINT8" => write!(w, "Some(crate::DataType::QInt8)")?,
+                "DT_QUINT8" => write!(w, "Some(crate::DataType::QUInt8)")?,
+                "DT_QINT32" => write!(w, "Some(crate::DataType::QInt32)")?,
+                "DT_BFLOAT16" => write!(w, "Some(crate::DataType::BFloat16)")?,
+                "DT_QINT16" => write!(w, "Some(crate::DataType::QInt16)")?,
+                "DT_QUINT16" => write!(w, "Some(crate::DataType::QUInt16)")?,
+                "DT_UINT16" => write!(w, "Some(crate::DataType::UInt16)")?,
+                "DT_COMPLEX128" => write!(w, "Some(crate::DataType::Complex128)")?,
+                "DT_HAFL" => write!(w, "Some(crate::DataType::Hafl)")?,
+                "DT_UINT32" => write!(w, "Some(crate::DataType::UInt32)")?,
+                "DT_UINT64" => write!(w, "Some(crate::DataType::UInt64)")?,
+                _ => panic!("{} is not supported", t.descriptor().name()),
+            },
+            Some(AttrValue_oneof_value::shape(shape)) => {
+                let dims: Vec<_> = shape
+                    .get_dim()
+                    .iter()
+                    .map(|x| format!("{}", x.get_size()))
+                    .collect();
+                if dims.is_empty() {
+                    write!(w, "None")?;
+                } else {
+                    write!(w, "Some(crate::Shape::from(&[{}])", dims.join(", "))?;
+                }
+            }
+            Some(AttrValue_oneof_value::tensor(tensor)) => {
+                dbg!(tensor);
```

Comment:
There's a few debug statements that need to be removed.

Reply:
`dbg!`s were removed.